internal/dao/log_item.go

```diff
@@ -89,9 +92,25 @@ func (l *LogItem) Render(paint string, showTime bool, bb *bytes.Buffer) {
 		bb.WriteString("[-::] ")
 	}
 
-	if index > 0 {
-		bb.Write(l.Bytes[index+1:])
-	} else {
-		bb.Write(l.Bytes)
+	bb.Write(colorizeJSON(l.Bytes[index+1:], showJson))
+}
+
+func colorizeJSON(text []byte, showJson bool) []byte {
+	if !showJson {
+		return text
+	}
+	var obj map[string]interface{}
+	err := json.Unmarshal(text, &obj)
+	if err != nil {
+		return text
+	}
+	f := colorjson.NewFormatter()
+	f.Indent = 0
+	f.ObjectSeparator = false
+
+	s, err := f.Marshal(obj)
+	if err != nil {
+		return text
```

Comment:
I would like to see `err` in debug logs, but I don't know how... Here and at the line 105

Reply:
I can kind-of see why that might be useful, but if the text presented cannot be unmarshalled into a json object, then it’s not likely to be json. What more is there to say? Otherwise I fear it will just clog up the logs. Nevertheless I will have a look to see if it’s possible.

Reply:
I see only 1 use case: your application generates json-like log but with some problems. And in this case, you will see un-colored json and you can create an issue here, but the real problem is in your app :) Okay, I agree with you.